Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ruby:3.2.2
+
+RUN apt-get update  && \
+    apt-get install -y ca-certificates curl gnupg  &&\
+    mkdir -p /etc/apt/keyrings && \
+    curl -fsSL https://deb.nodesource.com/gpgkey/nodesource-repo.gpg.key | gpg --dearmor -o /etc/apt/keyrings/nodesource.gpg && \
+    NODE_MAJOR=20  && \
```

Comment:
Remove one of the excess spaces before &&.

Reply:
Also, you didn't like my version in which NODE_MAJOR was an ARG?

Reply:
I just copied the code from the website

Reply:
Well, you could have copied the code from my comment. :)

But whatever. It's not that important.